### PR TITLE
clarify tablets

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
cc @Hixie

File: faq.md [hunk 1]
```diff
@@ -371,9 +371,12 @@ iPhone models), and ARM Android devices.
 We support developing Flutter apps with Android and iOS devices, as
 well as with Android emulators and the iOS simulator.
 
-We test on a variety of low-end to high-end phones (excluding tablets)
+We test on a variety of low-end to high-end phones
 but we don't yet have an official device compatibility guarantee.
-We do not offer support for tablets or have tablet-aware layouts.
+
+We do not test on tablets, so there may be issues with some
```
Comment (Hixie) [inline faq.md]: yet?

**Comment by Hixie** [comment]
LGTM, but travis is angry.

**Comment by sethladd** [comment]
Deploy errors for staging. Will land.

**Pull request merged**